icinga-app/app.manifest

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<asmv1:assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv1="urn:schemas-microsoft-com:asm.v1" xmlns:asmv2="urn:schemas-microsoft-com:asm.v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <assemblyIdentity version="1.0.0.0" name="Icinga 2"/>
```

Comment:
[Documentation](https://learn.microsoft.com/en-us/windows/win32/sbscs/application-manifests#assemblyidentity) for `name`:
> Uniquely names the application or assembly. Use the following format for the name: Organization.Division.Name. For example Microsoft.Windows.mysampleApp. Required.

So doesn't look like a name for human consumption but rather like a scoped identifier. Also not sure if you're supposed to use spaces in there. There's also an [example](https://learn.microsoft.com/en-us/windows/win32/sbscs/application-manifests#msix) with a 2 level name, so my suggestion for the value would be `Icinga.Icinga2`.